libsolidity/ast/Types.cpp

```diff
@@ -3323,13 +3323,13 @@ Type const* FunctionType::selfType() const
 	return m_parameterTypes.at(0);
 }
 
-ASTPointer<ASTString> FunctionType::documentation() const
+ASTPointer<StructuredDocumentation> FunctionType::documentation() const
```

Comment:
We should maybe change this later so that it returns a plain pointer to StructuredDocumentation. I don't see why we should create a new shared_ptr below in the failure case.